### DISCUSSION TITLE
Support Generic OIDC Auth

### DISCUSSION THREAD

**Discussion by hugosxm** [heyform/heyform#58] [Ideas]
Hi,

It would be cool to be able to use generic OIDC providers instead of just google or apple, I think about authelia, authentik and keycloak ;)

Thanks !

**Comment by MitPitt** [comment]
+1 OIDC is great, I'm using authentik

**Comment by HackerBuddy229** [comment]
I would love to see some kind of OIDC (or even LDAP) support. I am currently migrating from Formbricks since they removed OIDC and a bunch of features from community edition. Heyforms looks to be a really attractive option, although the lack of private SSO would make it difficult to adopt.

**Comment by cduchenoy** [comment]
I've submitted PR https://github.com/heyform/heyform/pull/258 which adds generic OIDC support via openid-client, allowing HeyForm to authenticate against any standards-compliant OpenID Connect provider (Authelia, Keycloak, Authentik, etc.).

This has been tested with Authelia as the identity provider.

**Comment by HectorBenito** [comment]
Whats the status of this feature, heyforms is very good app but lacks oidc right now wich is essential for a modern app
